Source/ModuleKISInventory.cs

```diff
@@ -1375,7 +1391,9 @@ public enum InventoryType {
     GUILayout.EndHorizontal();
 
     if (contextItem == null) {
-      GUI.DragWindow(new Rect(0, 0, 10000, 30));
+      Width = (int)Math.Round(10000 * GameSettings.UI_SCALE * GameSettings.UI_SCALE_APPS);
```

Comment:
Width don't need to be scaled. Simply set it to a higher value, like `100,000`. It's clipped by the windows client rect anyway.

Reply:
I wasn't sure what was going on here and just followed the pattern I'd been using on the other dimensions.